### PR TITLE
Feature/email escalation

### PR DESCRIPTION
**Pull request by dadamsretrorabbit**
Requests Merge of my branch that implements a Mailing List and email escalation logic.

### CODE REVIEW

File: EmailNotifications/Email.cs [hunk 1]
```diff
@@ -38,72 +39,111 @@ private static void SendCompletedCallback(object sender, AsyncCompletedEventArgs
...
+                        }
+
+                        message.Subject = "WaterLog Notification";
+                        message.IsBodyHtml = true;
+                        message.Body = email;
+                        client.Send(message);
+                    }
+                    catch (Exception e)
+                    {
+                        Console.WriteLine("{0} Exception caught.", e);
```
Comment (FredJEhlers) [inline EmailNotifications/Email.cs]: Console logs

File: EmailNotifications/Email.cs [hunk 2]
```diff
@@ -38,72 +39,111 @@ private static void SendCompletedCallback(object sender, AsyncCompletedEventArgs
...
-                                    "R " + values[4], "Has been lost",values[6]+"L or R "+ values[5]," is being lost!", "per hour","Resolve issue or precess",values[7],"call third party help: 011111929292"};
-                foreach (var alert in items)
+                string[] fontSizeProperties = { "6", "3", "3", "6", "4", "6", "4", "4", "4", "4", "2" };
+                System.Text.StringBuilder sb = new System.Text.StringBuilder();
+                using (TableStructure.Table table = new TableStructure.Table(sb, id: "some-id", align: "center"))
                 {
-                    using (var tr = table.AddRow(classAttributes: "someattributes"))
+                    table.StartBody();
+                    int count = 0;
+                    string[] items = { "<b>" + values[0] + " has a " + values[1] + "</b>", "<b>("+ values[2]+")</b>","This problem has been <b>unresolved for " + values[3]+" days</b>",
```
Comment (FredJEhlers) [inline EmailNotifications/Email.cs]: split over multiple lines

File: WaterLog_Backend/Procedures.cs [hunk 3]
```diff
@@ -186,26 +255,26 @@ public Boolean isLeakage(double first, double second)
...
+            try
+            {
+                var leaks = _db.SegmentLeaks;
+                var leak = leaks
+                .Where(sudo => sudo.SegmentsId == sectionid)
+                .Single();
 
-            string[] template = { "Segment " + leak.SegmentsId, getSegmentStatus(leak.SegmentsId), leak.Severity, getLeakPeriod(leak), calculateTotalCost(leak).ToString(), calculatePerHourCost(leak).ToString(), calculateLitresPerHour(leak).ToString(), buildUrl(leak.SegmentsId) };
-            return template;
+                string[] template = { "Segment " + leak.SegmentsId, getSegmentStatus(leak.SegmentsId), leak.Severity, getLeakPeriod(leak), Math.Round(calculateTotalCost(leak)).ToString(), Math.Round(calculatePerHourCost(leak)).ToString(), Math.Round(calculateLitresPerHour(leak)).ToString(), buildUrl(leak.Id) };
```
Comment (FredJEhlers) [inline WaterLog_Backend/Procedures.cs]: Split over multiple lines

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: EmailNotifications/Email.cs (hunk 2 above)
Comment (dadamsretrorabbit) [reply EmailNotifications/Email.cs]: Split the line over multiple lines.

File: WaterLog_Backend/Procedures.cs (hunk 3 above)
Comment (dadamsretrorabbit) [reply WaterLog_Backend/Procedures.cs]: Split the line over multiple lines.

File: EmailNotifications/Email.cs (hunk 1 above)
Comment (dadamsretrorabbit) [reply EmailNotifications/Email.cs]: Threw an exception instead of Console.Writeline.

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**